app/components/Header/LockingState/components/SafesUnlocked.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+
+import styles from 'assets/css/global.css'
+
+class SafesUnlocked extends Component {
```

Comment:
This is the component that renders the countdown when the account is unlocked ;)